recipe/meta.yaml

```diff
@@ -1,36 +1,40 @@
 {% set name = "pydantic-core" %}
-{% set version = "0.42.0" %}
+{% set version = "2.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydantic_core-{{ version }}.tar.gz
-  sha256: 25d89d9c09c1e79d415e2150c11117c35115f9d08dea452061f3391d458ee61b
+  sha256: f9fffcb5507bff84a1312d1616406cad157806f105d78bd184d1e6b3b00e6417
 
 build:
   script:
     # PyPy has weird sysconfigdata name
     - rm -f $PREFIX/lib/pypy$PY_VER/_sysconfigdata.py  # [build_platform != target_platform and target_platform == "linux-ppc64le"]
     - {{ PYTHON }} -m pip install . -vv
   number: 0
+  skip:
+    # 
```

Comment:
Please make an upstream issue about this.

Reply:
https://github.com/pydantic/pydantic-core/issues/724